views/scss/qti/_choice.scss

```diff
@@ -27,11 +27,11 @@
             right: $elimCbOffsetLeft * 1px;
         }
         &:after {
-            content: "0";
+            content: '0';
```

Comment:
Since we will have to backport this, I would advise to avoid refactoring of code style and only push necessary changes.

Reply:
These are changes made by prettier, they won't hurt. We have an agreement to update files we touch in old repos with the code style changes

Reply:
It is ok to do this changes, but maybe in a separate commit or PR, just to avoid backporting conflicts. But if we think it wont be an issue, that is ok :)